flake.nix

```diff
@@ -17,6 +17,17 @@
       hci-effects = hercules-ci-effects.lib.withPkgs pkgs;
     in
     {
+      apps."${system}" = {
+        live = rec {
+          type = "app";
+          script = pkgs.writers.writeBash "emanotePlutonomiconLiveReload.sh" ''
+            set -xe
+            export PORT=7071
```

Comment:
Does `nix run` pass the user environment? If so, we would want to use it, and fallback to 7071 only if no $PORT is set. It allows user to change the port number if they desire.

Reply:
Not critical for this PR though.

Reply:
`PORT` seems to be a bit general. Can you not make Emanote take `EMANOTE_PORT` instead? I can imagine `PORT` being set by a lot of things, for whatever reason.

Reply:
> I can imagine `PORT` being set by a lot of things, for whatever reason.

Can you elaborate?

Reply:
Or better, I should make it take it as argument, along with `$HOST`: `emanote run -b 127.0.0.1:7071`

Reply:
I've implemented this in 6bbff83cee2e57c77aa7e5431f5055bbe9b8f211

Reply:
> Or better, I should make it take it as argument, along with `$HOST`: `emanote run -b 127.0.0.1:7071`

Done in https://github.com/srid/emanote/commit/cb8c989bd39563ad14cb569f1c9e272efd5c5c31